ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -238,18 +273,23 @@ + (BOOL)verifyHashForDiffUpdate:(NSString *)finalUpdateFolder
                    expectedHash:(NSString *)expectedHash
                           error:(NSError **)error
 {
+    NSError *localError;
     NSMutableArray *updateContentsManifest = [NSMutableArray array];
     [self addContentsOfFolderToManifest:finalUpdateFolder
                              pathPrefix:@""
                                manifest:updateContentsManifest
-                                  error:error];
-    if (*error) {
+                                  error:&localError];
+    if (localError && error) {
```

Comment:
I think we should return `NO` if the `localError` is set, regardless of whether the `error` is nil?

If you don't like that, feel free to modify the methods in this file to return boolean flags indicating success, since this file doesn't contain public documented APIs.